tests/hardware_tests/test_spot_to_aria_nav.py

```diff
@@ -0,0 +1,107 @@
+import sys
+
+import numpy as np
+from spot_rl.envs.nav_env import construct_config_for_nav
+from spot_rl.envs.skill_manager import SpotSkillManager
+from spot_rl.utils.utils import get_default_parser, map_user_input_to_boolean
+
+
+def parse_arguments(args=sys.argv[1:]):
+    parser = get_default_parser()
+    # parser.add_argument(
+    #     "-g",
+    #     "--goal",
+    #     help="input:string -> goal x,y,theta in meters and radians obtained from aria",
+    #     required=False,
+    #     type=str,
+    # )
+    parser.add_argument(
+        "-pct",
+        "--pick_target",
+        help="input:string -> pick target poplated from Aria",
+        required=True,
+        type=str,
+    )
+    parser.add_argument(
+        "-ot",
```

Comment:
...
@tusharsangam  what is the purpose of this file?
We don't have a mechanism to use aria in Hardware CI tests. 

If this is not needed, please delete the file.

Reply:
Moved to experiements/skill_test